pkg/crc/machine/machine.go

```diff
@@ -559,7 +559,7 @@ func (client *client) Status() (*ClusterStatusResult, error) {
 		return nil, errors.Wrap(err, "Cannot get root partition usage")
 	}
 	return &ClusterStatusResult{
-		CrcStatus:        vmStatus,
+		CrcStatus:        state.Running,
```

Comment:
I would mention in the commit log that this change is made possible by one of the previous commits.

Reply:
done